packages/ui5-proxy-middleware/src/base/utils.ts

```diff
@@ -150,17 +145,13 @@ export const getYamlFile = (args: string[]): string => {
  * @returns Path to the webapp folder
  */
 export const getWebAppFolderFromYaml = async (ui5YamlPath: string): Promise<string> => {
-    let webAppFolder = 'webapp';
-
     if (existsSync(ui5YamlPath)) {
-        const ui5Yaml = parseDocument(await promises.readFile(ui5YamlPath, { encoding: 'utf8' })).toJSON();
-
-        if (ui5Yaml.resources?.configuration?.paths?.webapp) {
-            webAppFolder = ui5Yaml.resources.configuration.paths.webapp;
-        }
+        const test = readFileSync(ui5YamlPath, { encoding: 'utf8' });
```

Comment:
Why did you switched to the `sync` function?

Reply:
Thank you for the question! 
I tried to find out if there are any known advantages or disadvantages for one or the other approach by searching the internet and by discussing with @nlunets . Conclusion: if you have a use case where you want to read two or more files in parallel, using the `async` version, obviously is the better more flexible option. However, if you read a single file and force synchronization by using `await`, there is no advantage of using a promise. And if two things have no advantage or disadvantage compared to each other, then I always prefer going for the simpler one, which is the `readFileSync`.
Not important but also for consistency, it felt right to use `readFileSync` because we already use `existsSync`.

Reply:
Thank you for the thorough explanation!